staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -52,114 +52,6 @@ func NewTopCmdPrinter(out io.Writer) *TopCmdPrinter {
 	return &TopCmdPrinter{out: out}
 }
 
-type NodeMetricsSorter struct {
```

Comment:
Move sorter out of `metrics_printer` to `metrics_sorter` because it clutters the printer file